object_analytics_node/CMakeLists.txt

```diff
@@ -195,6 +214,11 @@ install(TARGETS
   DESTINATION lib/${PROJECT_NAME}
 )
 
+install(TARGETS
```

Comment:
would you please notice the example usage of the tool somewhere? in readme or usage code?

Reply:
This will be addressed in readme once the code got merged.